plugins/plugin-github/package.json

```diff
@@ -6,10 +6,12 @@
 	"types": "dist/index.d.ts",
 	"dependencies": {
 		"@elizaos/core": "^0.1.9",
+		"@realityspiral/shared-sentry": "workspace:*",
```

Comment:
instead of shared-sentry can we call it @realityspiral/sentry

Reply:
updated